elastic.js/index.d.ts

```diff
@@ -6765,7 +6765,7 @@ declare module elasticjs {
     /*
      Allows you to set a specified filter on this request object.
      */
-    filter(filter: Object): Request;
+    filter(filter: Object | Filter): Request;
```

Comment:
I was getting type errors when trying to pass in a Filter. It works for me when it's just filter(filter: Filter) but I wasn't sure if there were other scenarios where removing Object would be an issue. If not, I can remove the Object type.